static/sidebar/js/modal/index.js

```diff
@@ -143,19 +143,24 @@
     },
 
     removeClass = function(){
-      for (var i = elements.length; i--;) {
-        elements[i].classList.remove('active');
-        arrow.classList.remove('active');
+      for (var i = 0; i < classNames.length; i++) {
+        var activeElements = document.getElementsByClassName(classNames[i]);
+        for (var j = 0; j < activeElements.length; j++) {
+          activeElements[j].classList.remove('active');
+          arrow.classList.remove('active');
+        }
       }
     };
 
     for (var i = 0; i < classNames.length; i++) {
-      elements = holder.getElementsByClassName(classNames[i]);
+      elements = document.getElementsByClassName(classNames[i]);
       for (var j = 0; j < elements.length; j++) {
         elements[j].addEventListener('click', pkTrack, false);
         elements[j].onmouseover = function(e){
           clearTimeout(TimeOut);
-          showPopUp(e);
+          if (e.target.className === "tvp-product-image") {
```

Comment:
Please change this from `e.target.className` to `e.target.classList.contains('tvp-product-image')` or remove it, `showPopUp` is already protecting itself for this